### DISCUSSION TITLE
Landscape Version?

### DISCUSSION THREAD

**Discussion by enomis123** [kudrykv/latex-yearly-planner#24] [Ideas]
Hi - would it be very complicated to create a landscape version of the planner ? TIA.

**Comment by kudrykv** [comment]
It would require creating a set of new configurations, LaTeX files, and likely redesigning stuff a bit.
Doable, although unlikely something that I'd do in the short term.

**Reply by enomis123** [reply]
I was afraid so . I will start with some trial-and-error. Let's see how far I will get... ;)

**Reply by enomis123** [reply]
Hi - I managed to created a landscape version (without quarterly and mos). My changes are quick&dirty and only valid for SN A5X so I‘d rather not share my code. But I can share my PDF if anyone is interested.

**Reply by kudrykv** [reply]
Please do 🙂
Someone may be interested in the solution!

**Comment by enomis123** [comment]
here we go.... planner in landscape ( with some limitations - no quarterly, no mos, daily schedule from 8 to 19 and Helvetic font) 
[sn_a5x.landscape.dailycal.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7918225/sn_a5x.landscape.dailycal.pdf)

**Reply by MrHappy** [reply]
I've just recently got my Remarkable and am looking for a landscape-version of a planner.  This looks good. Could you please share your code, so I can create one for 2023? (And tinker around in it ;) )..

**Reply by kudrykv** [reply]
Err
This repo is the code that generates the planner

**Reply by MrHappy** [reply]
Does it support landscape already? I figured you hadn't done this yet... Was I wrong to assume that?

**Reply by kudrykv** [reply]
You're right, I didn't do that yet (and my refactor is going slower than I expected, _sigh_). Maybe I've misunderstood you; you were asking to share the code. This repository generates planners. In the current state, though, you'll need to tinker with its templates.

**Reply by MrHappy** [reply]
I was referring to @enomis123 's reply about his/her changes to the sourcecode ;)

**Reply by kudrykv** [reply]
Oh 😳😬

**Reply by MrHappy** [reply]
No problem 😁
